libcnb-data/src/buildpack/mod.rs

```diff
@@ -58,7 +59,7 @@ pub use version::*;
 /// ```
```

Comment:
The rustdocs for `BuildpackDescriptor`, `SingleBuildpackDescriptor` and `MetaBuildpackDescriptor` still use `<Option<toml::value::Table>>>`.

Now there is a default, that could be removed too. Also, whilst perhaps out of scope for this PR, I wonder if we need an example showing how to use them with a custom metadata type?

Reply:
Changed to use the default.